scripts/benchmark.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+go test -v -bench=. $(find . -iname '*storage*test.go' -not -path '/vendor/') -run=^$
```

Comment:
why not just `go test ./... -bench=. -run=^$` -- vendor gets excluded automatically, and we test all benchmarks since this is a global Makefile command.

Reply:
That ran benchmark against all packages thought it didn't have any benchmark files, and this output was much better :)